src/iexamine.cpp

```diff
@@ -4255,6 +4325,45 @@ void smoker_activate( player &p, const tripoint &examp )
     add_msg( _( "You light a small fire under the rack and it starts to smoke." ) );
 }
 
+void iexamine::mill_finalize( player &, const tripoint &examp, const time_point &start_time )
+{
+    const furn_id cur_mill_type = g->m.furn( examp );
+    furn_id next_mill_type = f_null;
+    if( cur_mill_type == f_wind_mill_active ) {
+        next_mill_type = f_wind_mill;
+    } else if( cur_mill_type == f_water_mill_active ) {
+        next_mill_type = f_water_mill;
+    } else {
+        debugmsg( "Furniture executed action mill_finalize, but is of type %s",
+                  g->m.furn( examp ).id().c_str() );
```

Comment:
Don't think this is needed here
```suggestion
                  g->m.furn( examp ).id() );
```